app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -217,10 +221,10 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
   }
 
   fun hideSyncIndicator() {
-    //TODO handle visibility
```

Comment:
We should toggle visibility changes with animation. A subtle fade in/out will look good.